core.lua

```diff
@@ -1420,6 +1422,16 @@ do
 		end
 	end
 
+	function AddLegionScore(tooltip, profile)
+		if profile.legionScore and profile.legionScore > 0 then
```

Comment:
I think the issue here would be if the character didn't have a legion score then it wouldn't show the legion main score. We should just use the main score if it exists and is higher than their legionScore.

I don't remember if we're guarded against nil in that check there, but if so I imagine the logic like:

```
if profile.legionMainScore > profile.legionScore then
  -- show legion main's score
elseif profile.legionScore > 0 then
  -- show legion score
end
```

Reply:
Oops, true, forgot about that !